### PR TITLE
Swift 2.0 changes

### PR DESCRIPTION
**Pull request by alexanderjarvis**
Includes fixes from the migration tool + other xcode things + something to make it actually compile instead of seg fault due to correct function sigs.

### CODE REVIEW

File: PeerKit/PeerKit.swift [hunk 1]
```diff
@@ -111,22 +111,27 @@ public func stopTransceiving() {
...
 
-public func sendEvent(event: String, object: AnyObject? = nil, toPeers peers: [MCPeerID]? = session?.connectedPeers as? [MCPeerID]) {
-    if peers == nil || (peers!.count == 0) {
-        return
-    }
-    var rootObject: [String: AnyObject] = ["event": event]
-    if let object: AnyObject = object {
-        rootObject["object"] = object
+public func sendEvent(event: String, object: AnyObject? = nil, toPeers peers: [MCPeerID]? = session?.connectedPeers) {
+    if let peers = peers where !peers.isEmpty {
```
Comment (jpsim) [inline PeerKit/PeerKit.swift]: `guard`?

**Comment by jpsim** [comment]
Thanks for this, @alexanderjarvis! Everything here looks great, minus the one place where there could be a `guard` rather than the big `if let`.

**Comment by alexanderjarvis** [comment]
Done!

I'm not 100% sure `guard` is a good thing :tm: because I'm used to "everything is an expression" from Scala. Sure it lets you prevent some nesting which is handy, but it feels more imperative than functional with the early exit and explicit return.

**Comment by jpsim** [comment]
Thanks!

**Pull request merged**